### PR TITLE
Update CRDs to the latest

### PR DESCRIPTION
**Pull request by hjiawei**
## Description

`make gen-files` and `make gen-versions`.

## For PR author

- [ ] Tests for change.
- [x] If changing pkg/apis/, run `make gen-files`
- [x] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by rene-dekker — APPROVED** [review]
From what I understand these AlertExceptions have never been a documented feature in a release version. For this reason removing a field from the CRD should not cause issues for any users.

LGTM

**Review by rene-dekker — COMMENTED** [review]
From what I understand these AlertExceptions have never been a documented feature in a release version. For this reason removing a field from the CRD should not cause issues for any users.

File: pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml [hunk 1]
```diff
@@ -76,8 +76,6 @@ spec:
                 items:
                   type: string
                 type: array
-            required:
```
Comment (caseydavenport) [inline pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml:79]: Was this done manually? I don't see a corresponding CRD go code update

File: pkg/crds/enterprise/crd.projectcalico.org_alertexceptions.yaml [hunk 2]
```diff
@@ -37,25 +37,30 @@ spec:
...
+                description: StartTime defines the start time from which this alert
+                  exception will take effect. If the value is in the past, matched
+                  alerts will be filtered immediately. If the value is changed to
+                  a future time, alert exceptions will restart at that time.
+                format: date-time
+                type: string
             required:
             - description
             - selector
+            - startTime
```
Comment (caseydavenport) [inline pkg/crds/enterprise/crd.projectcalico.org_alertexceptions.yaml:60]: Introducing a new required field is a bit scary - old resources that used to apply cleanly won't apply any more. Is this really what we want?

**Comment by caseydavenport** [comment]
> From what I understand these AlertExceptions have never been a documented feature in a release version. For this reason removing a field from the CRD should not cause issues for any users.

@rene-dekker unfortunately documentation isn't the deciding factor in whether or not a feature exists, as much as we might like it to be. The API and CRDs are self-documenting, meaning if it's in a release then the feature exists.

File: pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml (hunk 1 above)
Comment (hjiawei) [reply pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml:79]: No. Not manual.

File: pkg/crds/enterprise/crd.projectcalico.org_alertexceptions.yaml (hunk 2 above)
Comment (hjiawei) [reply pkg/crds/enterprise/crd.projectcalico.org_alertexceptions.yaml:60]: We haven't officially released this yet (though in the release already) so there should be no old resources in the cluster.

File: pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml (hunk 1 above)
Comment (caseydavenport) [reply pkg/crds/operator/operator.tigera.io_amazoncloudintegrations.yaml:79]: Ah, I see. sounds good.

**Pull request merged**